source/base/mpi.cc

```diff
@@ -580,11 +384,6 @@ namespace Utilities
         return (my_destinations.size() == n_destinations);
       }();
 
-      // If all processes report that they have unique destinations,
-      // then we can short-cut the process using a consensus algorithm (which
```

Comment:
Please leave these lines untouched and just remove the text in parentheses.

Reply:
Done! I will merge all commits when we are done!